src/styles/index.scss

```diff
@@ -1,3 +1,138 @@
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
+}
+
+.block-card {
+  width: 198px;
+  height: 406px;
+  flex-shrink: 0;
+  flex-direction: column;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.block-card__image {
+  margin: 32px 19px 36px;
+}
+
+.image-product {
+  width: 160px;
+  height: 134px;
+}
+
+.name-product {
+  width: 166px;
+  height: 36px;
+  margin: 0 16px 4px;
+
+  &-text {
+    color: #060B35;
+    font-size: 12px;
+    font-style: normal;
```

Comment:
font-style: normal is a default value, no need to copy all styles from the mockup